examples/avmnist_multimodal/model.py

```diff
@@ -0,0 +1,52 @@
+"""
+Define the Multimodal Deep Learning (MMDL) model and configure model hyperparameters.
+References: 1. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_simple_late_fusion.py
+            2. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_low_rank_tensor.py
+            3. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_multi_interac_matrix.py
+"""
+
+from kale.embed.feature_fusion import Concat, LowRankTensorFusion, MultiplicativeInteractions2Modal
```

Comment:
A better name for MultiplicativeInteractions2Modal

Reply:
changed it to  "bimodal_matrix_fusion_interactor"

Reply:
To discuss the appropriateness of "interactor". Have a prelim discussion with @shuo-zhou about this maybe as he may have some good ideas.